### PR TITLE
Update cris to cris-fsr in NCAR converter

### PR DESCRIPTION
**Pull request by ashley314**
## Description

Updating `cris` to `cris-fsr` for the NCAR's obs2ioda converter. 

## Issue(s) addressed

Addresses https://github.com/JCSDA-internal/skylab/issues/327

## Dependencies

List the other PRs that this PR is dependent on:
- [x] https://github.com/JCSDA-internal/SIMOBS/pull/235
- [x] https://github.com/JCSDA-internal/skylab/pull/341
- [x] https://github.com/JCSDA-internal/ewok/pull/929

## Impact

Changes the observation ingest suite, so we will test with that CI.

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] I have run the unit tests before creating the PR

### CODE REVIEW

File: src/ncar-bufr2nc-fortran/define_mod.f90 [hunk 1]
```diff
@@ -343,7 +343,7 @@ subroutine set_name_sensor(instid, sensor)
       case (297); sensor = 'ahi'
       case (207); sensor = 'seviri'
       case (420); sensor = 'airs'
-      case (620); sensor = 'cris'
+      case (620); sensor = 'cris-fsr'
```
Comment (BenjaminRuston) [inline src/ncar-bufr2nc-fortran/define_mod.f90:346]: this seems to be not flexible for `cris` and `cris-fsr` which changed a specific crossover date

**Comment by BenjaminRuston** [comment]
ok @ashley314 this is essentially hardcoded to the `cris-fsr`

it is a hard date crossover where NOAA switched processing, unfortunate as the sensor ID seems to be the same 
`      case (620); sensor = 'cris-fsr'`

**Comment by ashley314** [comment]
> ok @ashley314 this is essentially hardcoded to the `cris-fsr`
> 
> it is a hard date crossover where NOAA switched processing, unfortunate as the sensor ID seems to be the same ` case (620); sensor = 'cris-fsr'`

@BenjaminRuston When you say "sensor ID" are you referring to `case (620)`? I did not know how to check that so what I have here was a guess and it seemed to work

**Comment by BenjaminRuston** [comment]
@ashley314 to be thorough we can get a `cris` file before the change to `cris-fsr` and see if it to is caught by the `case(620)`

if it has a unique identifier that would be great, but I'm guessing they may very well be the same

**Comment by ashley314** [comment]
@BenjaminRuston I was able to convert a CrIS file from 2020-01-01 using this converter. But that file seems to contain both the old CrIS and CrIS-fsr. For our skylab purposes, I don't think files older than that will need to be converted

File: src/ncar-bufr2nc-fortran/define_mod.f90 (hunk 1 above)
Comment (climbfuji) [reply src/ncar-bufr2nc-fortran/define_mod.f90:346]: Can we address this question/concern first before I review the set of PRs?

Comment (ashley314) [reply src/ncar-bufr2nc-fortran/define_mod.f90:346]: Yes. @BenjaminRuston I commented below, that I was able to convert a CrIS file from 2020-01-01 using this converter. But that file seems to contain both the old CrIS and CrIS-fsr. For our skylab purposes, I don't think files older than that will need to be converted. Do you agree or have concerns with that?

Comment (BenjaminRuston) [reply src/ncar-bufr2nc-fortran/define_mod.f90:346]: again this converter will hopefully deprecate and we'll move to the EMC one so yes agree for now let's move forward and cross idea of handling older CrIS data when we get our re-analysis funding

**Comment by PatNichols** [comment]
@ashley314 Do we have to merge this with the other PRs or is it independent?

File: src/ncar-bufr2nc-fortran/define_mod.f90 (hunk 1 above)
Comment (BenjaminRuston) [reply src/ncar-bufr2nc-fortran/define_mod.f90:346]: these numbers for obserrors are `431` long which is `fsr` we would need another vector for the `cris` data for truncated anyway.. 

in short this is hardcoded to `fsr` so this should be fine

**Review by BenjaminRuston — APPROVED** [review]
this is correct as this decoder is hardcoded in the vector to cris-fsr

**Comment by ashley314** [comment]
@PatNichols It should be merged with the other PRs. Thanks for checking!

**Comment by smaticka** [comment]
When I ran the `ingest-observations.yaml` after pulling this branch on `iodaconv` and in `ewok` and then running `make`, it completes the ingest experiment. but when I did the same thing, but also pull the branch for `skylab`, I get an error. I'll write it on the skylab PR.

**Comment by ashley314** [comment]
Thanks @smaticka, I followed up in that PR. There is a [line](https://github.com/JCSDA-internal/ewok/blob/bae54b6040ee6205fb506e8285cd717d6194d4e2/src/ewok/suites/ingestObservations.py#L32) in the `EWOK` PR and you shouldn't be getting that error. Can you check your `EWOK` repo and do a`git pull` to make sure the branch is up to date and matches what is in `feature/obsingest_cris-fsr`?

**Review by smaticka — APPROVED** [review]
Thanks @ashley314 . it worked for me when I pulled all 3 branches and ran `ingest-observations.yaml` for just `cris`

**Comment by cmgas** [comment]
If somebody can merge this I will merge the other two!

**Comment by climbfuji** [comment]
> If somebody can merge this I will merge the other two!

Ready whenever you are

**Comment by cmgas** [comment]
Merged!

**Pull request merged**